Makefile

```diff
@@ -86,7 +87,12 @@ deploy: manifests
 manifests: controller-gen
 	$(CONTROLLER_GEN) \
 		rbac:roleName=manager-role webhook \
-		crd:crdVersions=v1 \
+		crd:crdVersions=v1beta1,generateEmbeddedObjectMeta=true  \
+		output:crd:artifacts:config=config/crds/v1beta1 \
+		paths="./..."
+	$(CONTROLLER_GEN) \
+		rbac:roleName=manager-role webhook \
+		crd:crdVersions=v1,generateEmbeddedObjectMeta=true  \
```

Comment:
Why generate both v1 and v1beta1 CRDs? Do we need to support pre-1.16 k8s here?

Reply:
Nope, just missed that when rebasing. Didn't realise it had been removed from the upstream